js/click.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+var CarDealer = (function(oldCarDealer){
+  oldCarDealer.activateEvents = function(event){
+
+     var input = document.getElementById('input');
+     var inventory = document.getElementsByClassName('card');
+     var color = 0;
+     var section = document.getElementById('bodWrapper')
+
+     document.addEventListener("click", newClick);
+     input.addEventListener("keyup" , newData);
+              
+              function newClick(event){
+                for (var i=0; i<inventory.length; i++){
+                    var newSel = inventory[i];
+                    color = "#59BFBF";
+                    console.log(newSel)
+                    CarDealer.removeClass(newSel , color);
+
+                    if (event.target === inventory[i] || event.target.closest('div') === inventory[i]){
+                      color =  "#FFBDA3";
+                      CarDealer.addClass(newSel , color);
+                      input.focus();
+                      input.value = "";
+                    }
+                }
+            }
+
+          function newData(event) {
+             if(event.keyCode === 13) {
+                input.value = " ";
+              }
+
+            let newBio = document.getElementsByClassName('border');
+            newBio[0].childNodes[4].innerHTML = input.value;
```

Comment:
This works, but is very fragile. If one thing changes later in the HTML layout of your cards, your javascript will break. It's best to avoid that kind of coupling. If the section you want to replace has an id on it, you can use `getElementById` on `newBio[0]` just like you can on `document`. Try that and see if you can make it work.